src/schemas/file.js

```diff
@@ -12,50 +11,81 @@ export default {
 		},
 		parseDOM: [
 			{
-				tag: 'file',
+				tag: 'figure',
 				getAttrs: (node) => {
+					if (node.getAttribute('data-node-type') !== 'file') {
+						return false;
+					}
 					return {
 						url: node.getAttribute('data-url') || null,
-						fileName: node.getAttribute('data-fileName') || null,
-						fileSize: Number(node.getAttribute('data-fileSize')) || null,
```

Comment:
Why was the `Number()` cast removed here?

Reply:
`Number()` seems to have been an error in the first place. We use the [filesize package](https://www.npmjs.com/package/filesize) which returns a string such as `24kb` or `1Mb`.